### ISSUE TITLE
Python.import("numpy") -> EXC_BAD_ACCESS

### ISSUE THREAD

**Issue by matheusalano** [tensorflow/swift#259]
Hello, I'm trying to create a chatbot with swift and tf. I'm still trying to get started with tf but I'm facing a problem with Python interoperability. 

When I execute `Python.import("numpy")` I'm getting this error: `Thread 1: EXC_BAD_ACCESS (code=1, address=0x48)`

Here's my code:
```
init() {
        PythonLibrary.useVersion(3, 7)
        let np = Python.import("numpy")
        print(np)
        
        let url = URL(string: "/Users/matheusalano/Downloads/wiki.multi.pt.vec.txt")
        guard let file = try? String(contentsOfFile: url?.path ?? "", encoding: .utf8) else {
            fatalError("File not found.")
        }
        
        let rows = file.components(separatedBy: "\n").dropFirst()
        
        rows.enumerated().forEach { i, line in
            let splitLine = line.trimmingCharacters(in: .whitespaces).split(separator: " ", maxSplits: 1)
            let word = splitLine[0]
            let vect = splitLine[1]
            
//            let x = np.array(vect)
            
            exit(0)
        }
```

**Comment by matheusalano** [comment]
Here's what happened until now:
Xcode 10.3
Mojave 10.14.6

Xcode >> New Project >> macOS >> Command Line Tool.

Then I followed the instructions from `https://github.com/tensorflow/swift/blob/master/Installation.md`

After that, I got this error: `Library not loaded: @rpath/libswiftPython.dylib`. Fixed with the workaround found on #177

**Comment by rahulbhalley** [comment]
@matheusalano if the problem still persists for this error you can try [this](https://github.com/tensorflow/swift/issues/177#issuecomment-569477582) solution.
